modules/showheroes-bsBidAdapter.js

```diff
@@ -271,98 +188,49 @@ function createBids(bidRes, reqData) {
       bidUnit.vastUrl = bid.vastTag || bid.vastUrl;
     }
     if (bid.mediaType === BANNER) {
-      bidUnit.ad = getBannerHtml(bid, reqBid, reqData);
+      bidUnit.ad = getBannerHtml(bid, reqData);
```

Comment:
We already have the banner implementation in the viralize repo: https://github.com/showheroes/viralize/blob/master/viralize/web/prebid.py#L43

I honestly don't know how it's working, but that part is unchanged. In our roadmap, we have to work on banner support for our bidder because, as far as I know, it currently works: it will just load the player and start the waterfall.

Reply:
nobody is using the banner prebid and legacy showheroes is down. Let's remove the display part for now.